editor/editor_help.cpp

```diff
@@ -1782,7 +1782,7 @@ void EditorHelpBit::_bind_methods() {
 void EditorHelpBit::_notification(int p_what) {
 
 	if (p_what == EditorSettings::NOTIFICATION_EDITOR_SETTINGS_CHANGED) {
-		add_style_override("panel", EditorNode::get_singleton()->get_gui_base()->get_stylebox("ScriptPanel", "EditorStyles"));
+		add_style_override("panel", get_stylebox("ScriptPanel", "EditorStyles"));
```

Comment:
since the EditoHelpBit node is already part of the editor_node tree it is not necessary to call `get_stylebox` on the singleton. You can just call it on the node itself. I think it is only necessary when accessing style boxes in the initiator.
Correct me if I'm wrong.